authentication.go

```diff
@@ -23,7 +23,9 @@ type LoginInfo struct {
 	status   bool
 }
 
-type ValidateCallbackFunc func(username string, password string) (orgID string, intputID string, err error)
+type SetValuesFunc func(values map[string]interface{})
```

Comment:
Not sure whether value injection is a common practice in Golang or not 🤔
I can understand that `map` is a reference type in Golang, so this is safe. However, injecting values into parameters and returning nothing is not so straightforward to me.
I would suggest that the function `ValidateCallbackFunc` directly returns `(map[string] interface{}, error)` or even `(map[string] string, error)`; it may have a better readability.

Reply:
I am okay if @david74chou has no concerns about it.

Reply:
yeah, let's return the `map[string]interface{}` from `ValidateCallbackFunc` and insert each value into `client.Values`